docs/release-notes/release-notes-0.20.0.md

```diff
@@ -62,6 +62,10 @@ a certain amount of msats.
 - Added support for [P2TR Fallback Addresses](
   https://github.com/lightningnetwork/lnd/pull/9975) in BOLT-11 invoices.
 
+- A new experimental RPC endpoint
+  [XFindBaseLocalChanAlias](https://github.com/lightningnetwork/lnd/pull/10133)
```

Comment:
re my previous comment, if that is the desired behaviour, then we should warn here that the mappings are lost on restart